app/views/stats/subscriptions.html.erb

```diff
@@ -4,4 +4,28 @@
 </div>
 <div class="col-md-8 col-md-offset-1">
     <h2>Subscriptions</h2>
-</div>
+ 	<table class="table inline-grid">
+		<tr>
+			<thead>
+				<th>Subscriptions</th>
+				<th>Tags</th>
+			</thead>
+		</tr>
+		<% @tags.each do |map| %>
+		<% map.each do |subscriptions,tags| %>
+		<tr>
+			<tbody>
+				<td><%=subscriptions%></td>
+				<td>
+					<select style="width: 50%; height: 50px;">
+						<%tags.each do |tag| %>					      
+							<option><%=tag%></option>
```

Comment:
Then here, I actually think we could just display a list of tags, one per line in plain text, not in a select, so just the tag's name, a colon, and the true count next to it -- remember each of these is in the format `['tagname', 888]` :

```ruby
<p><%= tag[0] %>: <%= tag[1] %></p>
```

Reply:
Yes this works okay, the display is however a little off.

![new-stats](https://user-images.githubusercontent.com/17080976/54806112-1dc29500-4c8a-11e9-9865-d6021ccc72d7.gif)

Is there something I can do about this or should i just leave it at this?

Reply:
Huh, that is so weird. Let me look at the code, but I'm not sure why the `<p>` tags aren't making newlines. We could try `<br />` instead, between each line?

Reply:
No  no. I hadn't used` <p>` tags here.